integration/db/fixture.go

```diff
@@ -293,7 +294,7 @@ func SetupDatabaseTest(t *testing.T, options ...TestOptionFunc) *DatabasePack {
 	rcConf := servicecfg.MakeDefaultConfig()
 	rcConf.DataDir = t.TempDir()
 	rcConf.Auth.Enabled = true
-	rcConf.Auth.Preference.SetSecondFactor("off")
+	rcConf.Auth.Preference.SetSecondFactor(constants.SecondFactorOff)
```

Comment:
Taking the PR as-is, isn't this introducing a new usage of a deprecated constant?